### PR TITLE
More packages

### PR DESCRIPTION
**Pull request by MaximKraev**

### CODE REVIEW

**Comment by MaximKraev** [comment]
https://codio.myjetbrains.com/youtrack/issue/codio-7184

File: package_list.json [hunk 1]
```diff
@@ -15,7 +15,41 @@
...
-
+      },
+      "python2": {
+        "name": "Python2 tools set",
+        "description": "Set of tools for python2 development.",
+        "script": "python2/install.sh",
+        "version": "0.0.1",
+        "category": "web_development"
+      },
+      "python2": {
```
Comment (destitutus) [inline package_list.json]: `python3`

File: python2/playbook.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  sudo: true
+  tasks:
+    - apt: name=python state=present
+    - apt: name=python-dev state=present
+    - apt: name=python-pip state=present
+    - apt: name=python-setuptools state=present
+    - apt: name=python-numpy state=present
+    - apt: name=python-scipy state=present
+    - apt: name=python-tk state=present
+    - apt: name=ipython state=present
```
Comment (destitutus) [inline python2/playbook.yaml]: `build-essential` - need for building some dependencies

Comment (MaximKraev) [inline python2/playbook.yaml]: done

**Comment by destitutus** [comment]
👍

**Pull request merged**